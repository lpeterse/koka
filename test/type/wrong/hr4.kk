fun wrong(x) {
  (x : list<forall<a> a -> a>)
}