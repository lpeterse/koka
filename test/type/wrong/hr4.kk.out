test/type/wrong/hr4.kk(1,11): error: unannotated parameters cannot be polymorphic
 context : fun wrong(x) {
 (x : list<forall<a> a -> a>)
 }
 term : x
 inferred type: list<forall<a> (a) -> a>
 hint : annotate the parameter with a polymorphic type